src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -725,9 +728,17 @@ private static bool TryDeserializeJson<TResult>(string json, out TResult result)
                 }
                 return Task.FromResult<object>(null);
             });
+
+            var console = (Func<object, Task<object>>)(msg =>
+            {
+                _consoleLogger?.LogDebug((string)msg);
```

Comment:
Correct. These are user logs, nothing from the runtime, so we don't want them in the backend.